src/app/pages/ArticlePage/PagePromoSections/TopStoriesSection/TopStoriesItem/index.jsx

```diff
@@ -57,9 +57,9 @@ const TopStoriesItem = forwardRef(
           eventTrackingData={eventTrackingData}
         >
           <Promo.ContentWrapper>
-            {mediaType && <Promo.MediaIndicator />}
             <Title as={titleTag} script={script}>
               <Promo.Link>
+                {mediaType && <Promo.MediaIndicator />}
```

Comment:
Is moving this intentional?

Reply:
Hiya, yes it was. This was to achieve the desired look of the [UX designs](https://app.zeplin.io/project/5d4d77b1fcbfc79bf47e0294/screen/63a1b84f17c7a7162700a6c8), whereby the top stories section is wrapped in a rectangular box. Previously the media indicator was outside of the link, and so the link came into focus, the link would be pushed down underneath the media indicator. I know there will be some screen reader testing we need to do. If there are any other implications you can think of please let me know!

Reply:
Thanks for explaining! Makes sense.